installer/mix.exs

```diff
@@ -61,6 +61,7 @@ defmodule Phx.New.MixProject do
 
   def deps do
     [
+      {:hex_core, "~> 0.11.0"},
```

Comment:
Archives cannot include dependencies afaik :(

We will need to request against Hex API.

Reply:
reworked to use https://hex.pm/api/packages/phoenix instead. But as we need to parse JSON without a dependency, the whole check is gated behind an Elixir version check now.